vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -266,17 +266,48 @@ given by the `HierarchyNodes` parameter. This specifies a recursive hierarchy th
         <ReturnType Type="Edm.Boolean" Nullable="true" />
       </Function>
 
+      <Function Name="descendants" IsBound="true">
+        <Annotation Term="Core.Description" String="Returns the node values of descendants in the specified hierarchy of the given nodes with at most the specified distance" />
+        <Parameter Name="Nodes" Type="Collection(Edm.EntityType)" Nullable="false">
+          <Annotation Term="Core.Description" String="The given nodes" />
+        </Parameter>
+        <Parameter Name="NodePropertyPath" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Property path in the entity type of `Nodes` that addresses the node value" />
+        </Parameter>

```

Comment:
```suggestion
        <Parameter Name="Nodes" Type="Collection(Edm.String)" Nullable="false">
          <Annotation Term="Core.Description" String="Node values of the given nodes" />
        </Parameter>
```
I would much prefer the binding parameter to be a collection of _node values_ rather than a collection of _node entities_, from which the function must pick the node values property by following the `NodePropertyPath`. But, alas, such a collection of strings cannot be computed as a `rootExpr`: `$root/SalesOrganizations/ID` is not a `rootExpr`, because `/ID` is not a `collectionNavigation`.